### PR TITLE
Add controls for event title and date & time colors.

### PR DESCRIPTION
**Pull request by Camwyn**
Adds a control for the "event title" and "event date & time" color selections.

:camera: http://p.tri.be/8AVynK

[TEC-3685] && [TEC-3334]

[TEC-3685]: https://moderntribe.atlassian.net/browse/TEC-3685
[TEC-3334]: https://moderntribe.atlassian.net/browse/TEC-3334

Requires https://github.com/moderntribe/tribe-common/pull/1518

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -154,18 +154,61 @@ public function setup() {
...
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_title_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Title', 'the-events-calendar' ),
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:184]: :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'label'" and double arrow, but found 7.

<sub>hash: d0001d7a33392f772096f56658fea618</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -154,18 +154,61 @@ public function setup() {
...
+					'section'     => $section->id,
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:185]: :warning: Array double arrow not aligned correctly; expected 1 space(s) between "'section'" and double arrow, but found 5.

<sub>hash: 4377a39130adcfcb9ae6102fc098dcdb</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

**Review by lucatume — APPROVED** [review]
Actulally fine.

**Review by alliuca — APPROVED** [review]

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -154,18 +154,61 @@ public function setup() {
...
+				$manager,
+				$customizer->get_setting_name( 'event_date_time_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Date and Time', 'the-events-calendar' ),
+					'description' => esc_html__( 'Main date and time display on views and single event pages.', 'the-events-calendar' ),
+					'section'     => $section->id,
+				]
+			)
+		);
+
```
Comment (paulmskim) [inline src/Tribe/Customizer/Global_Elements.php:211]: Should all these come from the V2 customizer filter instead? If this is here, we show this on V1 right?

File: src/Tribe/Views/V2/Customizer.php [hunk 4]
```diff
@@ -32,6 +32,55 @@ public function filter_sections( array $sections, $customizer ) {
 		return $sections;
 	}
 
+	public function include_global_elements_settings( $section, $manager, $customizer ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Customizer.php:35]: :no_entry_sign: Missing doc comment for function include_global_elements_settings()

<sub>hash: 8a8099424b9b81b9338f9163d337ac1b</sub>

File: src/Tribe/Views/V2/Customizer.php [hunk 5]
```diff
@@ -32,6 +32,55 @@ public function filter_sections( array $sections, $customizer ) {
...
+
+		$manager->add_control(
+			new \WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_date_time_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Date and Time', 'the-events-calendar' ),
+					'description' => esc_html__( 'Main date and time display on views and single event pages.', 'the-events-calendar' ),
+					'section'     => $section->id,
+					'priority' => 8,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Customizer.php:78]: :warning: Array double arrow not aligned correctly; expected 4 space(s) between "'priority'" and double arrow, but found 1.

<sub>hash: 64bcc91a53b41c9c135527e88034c6e4</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 6]
```diff
@@ -881,6 +884,10 @@ public function filter_customizer_sections( $sections, $customizer ) {
 		return $this->container->make( Customizer::class )->filter_sections( $sections, $customizer );
 	}
 
+	public function action_include_global_elements_settings( $section, $manager, $customizer ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:887]: :no_entry_sign: Missing doc comment for function action_include_global_elements_settings()

<sub>hash: 7b79994eeff24447631e540381c690fa</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)
:warning: 1 warning(s)

**Review by paulmskim — APPROVED** [review]

**Pull request merged**